src/ObjectEncryptorFactory.php

```diff
@@ -207,13 +207,14 @@ private function addVersion2Wrappers($encryptor)
     }
 
     /**
+     * @param bool $createLegacyEncryptorIfAvailable
      * @return ObjectEncryptor Object encryptor instance.
      * @throws ApplicationException
      */
-    public function getEncryptor()
+    public function getEncryptor($createLegacyEncryptorIfAvailable = false)
```

Comment:
Tu asi nedava moc zmysel nastavovat to ako property tej factory triedy cez setter, ze? 
Lebo raz mozes chciet objekt s legacyEncryptorom a raz bez..

Reply:
Jo, to je presne to co nechci :) - presne - jednou to chci s a jednou bez - tady to ma podporovat aes sifru  https://github.com/keboola/job-queue-internal-api-php-client/blob/617a358667e775b28b03bc166b5514e3c608ee98/src/JobFactory/Job.php#L97 a tady uz ne https://github.com/keboola/job-queue-internal-api-php-client/blob/617a358667e775b28b03bc166b5514e3c608ee98/src/JobFactory/Job.php#L102